gitensite/apps/bookinfo/external.py

```diff
@@ -0,0 +1,58 @@
+import requests
+from cachecontrol import CacheControl
+from xml.etree import ElementTree
+
+request_session = requests.session()
+cached_session = CacheControl(request_session)
+
+#These functions take a Book object as an argument, and return either a URL if a result is found, or an empty string if there is no result
+
+def librivox_api(book):
```

Comment:
really need a test for this so that if (when) librivox changes its API, we'll notice if it breaks. Same for standard ebooks.

Reply:
I have added a test for this feature

Reply:
awesome!